types/acriosSystems/ACR-CV-101L-M-X/uplink.js

```diff
@@ -682,19 +682,44 @@ return {
 
 }
 
+function flattenData(data) {
+	const flatData = { ...data };
+	// delete flatData.data;
+
+	data.data.forEach((item, index) => {
+		if (item.dif && item.vif) {
+			const difVif = `${item.dif ? item.dif.join('_') : ""}__${item.vif ? item.vif.join('_') : ""}`;
+			flatData[`data_${index}_vif_dif`] = difVif;
```

Comment:
if possible try to stick to `camelCase` for dataKey naming

Reply:
done